docs/src/modules/components/Ad.js

```diff
@@ -12,7 +12,7 @@ const styles = (theme) => ({
   root: {
     position: 'relative',
     minHeight: 124,
-    maxWidth: 345,
+    maxWidth: '30ch',
```

Comment:
It's slightly bigger than before. The previous 345 seem arbitrary so I just went with a round value.